tests/units/utils/test_checkpoint.py

```diff
@@ -128,157 +127,6 @@ def track_and_save_best(
     )
 
 
-def _test_nans_checkpointing(
```

Comment:
Since I simplified the logic of the nans checkpointing, I don't think these tests are useful anymore.